specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/examples/Services_CheckStatus.json

```diff
@@ -16,14 +16,29 @@
           "diagnostic.echo",
           "diagnostic.error",
           "diagnostic.fastCount",
+          "DataMigration.AzureSqlDbPostMigrationValidationScenarioId",
           "DataMigration.CompleteAzureSqlDbMIMigration",
           "ConnectToSource.SqlServer",
           "GetUserTables.Sql",
+          "DataMigration.SelectSourceForSqlManagedInstanceScenario",
           "ConnectToTarget.AzureSqlDbMI",
           "ConnectToTarget.SqlDb",
-          "Migrate.SqlServer.SqlDb",
+          "ConnectToTarget.SqlServer",
+          "DataMigration.RunAssessment",
```

Comment:
Isnt DataMigration.RunAssessment a private scenario? I don't think it is exposed.

Also I dont see DataMigration.RunSqlManagedInstanceMigration, DataMigration.SelectTargetServerForSqlManagedInstanceScenario, DataMigration.SelectSourceForSqlManagedInstanceScenario in our AdvisorScenarioConstants.cs